### ISSUE TITLE
Implement a "Add Species Diagnosis" component as a dialog

### ISSUE THREAD

**Issue by BlakeDraper** [USGS-WiM/whispers#65]

**Comment by njestes** [comment]
Reopening. Strange issue where eventdetails request is not returning location_species id and species_diagnosis id, which are needed for editing. This was not happening earlier, so not sure what is going on. I am logged in as Whisper Admin, so based on what I'm seeing in the backend services code, I should be seeing the full serialized object with ids for all levels.

**Comment by BlakeDraper** [comment]
@aaronstephenson Was the exclusion of these IDs intentional, for the same reasons the Organization ID was excluded? If so, we need to reinstate them so we can event management CRUD working in the app.

**Comment by aaronstephenson** [comment]
After a little investigating, here is what I found:

First, the business rules clearly state that location_species_id and species_diagnosis_id must be hidden from public view.

![image](https://user-images.githubusercontent.com/8713712/43597507-e141e796-9647-11e8-8e9a-0809e12ba367.png)

![image](https://user-images.githubusercontent.com/8713712/43597536-fdda7f1c-9647-11e8-86cd-244b2be4b5aa.png)

And I confirmed that the `/eventdetails/` endpoint is indeed not returning those ID fields in the JSON response when the user is not logged in.

However, when I log in as Admin I do see those ID fields:

![image](https://user-images.githubusercontent.com/8713712/43598486-6ba034f4-964a-11e8-9274-c21db9f815a4.png)

Maybe it was just a fluke @njestes? Or maybe we're looking in different places?
